website/views.py

```diff
@@ -283,22 +284,32 @@ def dashboard(auth):
 @must_be_logged_in
 def watched_logs_get(**kwargs):
     user = kwargs['auth'].user
-    page_num = int(request.args.get('pageNum', '').strip('/') or 0)
-    page_size = 10
-    offset = page_num * page_size
-    recent_log_ids = itertools.islice(user.get_recent_log_ids(), offset, offset + page_size + 1)
+    try:
+        page = int(request.args.get('page', 0))
+    except ValueError:
```

Comment:
Disregard. I don't think there's a ways for the user to pass `None` in the querystring.